CHANGELOG.md

```diff
@@ -118,4 +118,5 @@ can get data for both organization and organization brand pages.
 * Updated `CommentAction.Attribute.value` to use AttributedEntity, to fix issue where share 
   requests with mentions aren't being persisted correctly.
 
-## 3.0.6 (work in progress)
+## 3.1.0 (Sep 7, 2022)
```

Comment:
Version 3.0.6 has been unintentionally released while getting the CI process working so the changelog needs to be updated to reflect what had been added in that version.

Reply:
Thanks for your heads up @MarcFletcher . Will update changelog when CircleCI is configured.